File: 58_Length_Last_Word/0ms.c
int lengthOfLastWord(char *s) {
  int length = 0;
  int n = strlen(s);
  int lastIndex = n - 1;

  while(lastIndex >= 0 && s[lastIndex] == ' ')
    lastIndex--;

  while(lastIndex >= 0 && s[lastIndex] != ' ') {
    length++;
    lastIndex--;
  }

  return length;
}

File: 58_Length_Last_Word/0ms.java
class Solution {
  public int lengthOfLastWord(String s) {
    int length = 0, lastIndex = s.length() - 1;

    while(lastIndex >= 0 && s.charAt(lastIndex) == ' ')
      lastIndex--;

    while(lastIndex >= 0 && s.charAt(lastIndex) != ' ') {
      length++;
      lastIndex--;
    }

    return length;
  }
}
